Terminal Commands.md

```diff
@@ -59,3 +59,12 @@ Some commands can be found in the `bin` directory.
 
 `-f` forcefully do a task i.e. without asking for confirmation<br/>
 `-r` recursively do a task(looks in subdirectories too)
+
+
+###### Vim commands:
```

Comment:
Will do it in an hour
On May 1, 2016 4:04 PM, "Akash Trehan" <notifications@github.com> wrote:

> In Terminal Commands.md
> <https://github.com/CodeMaxx/Terminal-Commands-Beginner/pull/2#discussion_r61684946>
> :
>
> > @@ -59,3 +59,12 @@ Some commands can be found in the `bin` directory.
> >
> >  `-f` forcefully do a task i.e. without asking for confirmation<br/>
> >  `-r` recursively do a task(looks in subdirectories too)
> > +
> > +
> > +###### Vim commands:
>
> ^this
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeMaxx/Terminal-Commands-Beginner/pull/2/files/665ec02c341d847ffe40e7d5848b84c1aeefda32#r61684946>
>